README.md

```diff
@@ -1,4 +1,10 @@
 # antsibull -- Ansible Build Scripts
+[![Python linting badge](https://github.com/ansible-community/antsibull/workflows/Python%20linting/badge.svg?event=push)](https://github.com/ansible-community/antsibull/actions)
+[![Python testing badge](https://github.com/ansible-community/antsibull/workflows/Python%20testing/badge.svg?event=push)](https://github.com/ansible-community/antsibull/actions)
+[![Build CSS testing badge](https://github.com/ansible-community/antsibull/workflows/Build%20CSS/badge.svg?event=push)](https://github.com/ansible-community/antsibull/actions)
+[![dumb PyPI on GH pages badge](https://github.com/ansible-community/antsibull/workflows/👷%20dumb%20PyPI%20on%20GH%20pages/badge.svg?event=push)](https://github.com/ansible-community/antsibull/actions)
```

Comment:
You may want to add some `branch=main` to the image URLs and maybe turn targets into something like https://github.com/ansible-community/antsibull/actions?query=workflow%3A%22Python+testing%22

Reply:
Thanks, I've adjusted the URLs.